module/admin/helper.js

```
@@ -135,4 +145,330 @@ module.exports = class adminHelper {
+            success: false,
+            message: messageConstants.apiResponses.INVALID_RESOURCE_TYPE,
+          };
+        }
+      } catch (error) {
+        return resolve({
+          success: false,
+          message: error.message,
+          data: false,
+        });
+      }
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)